main.py

```diff
@@ -264,6 +271,29 @@ def _get_user_quality_preference(self, stream_options):
             reversed(filtered_qualities)
         )
 
+    def stream_history(self):
+        """Starts streaming all streams that were playing when last session was closed."""
+        stream_history = self.load_stream_history()
+        if stream_history:
+            for stream in stream_history:
+                self.add_new_stream(stream)
+
+    def save_stream_to_history(self, url):
```

Comment:
Since we're writing to a file and this could get messy, let's leave this for now.